src/Exporter/ZipkinExporter.php

```diff
@@ -9,41 +9,47 @@
 
 /**
  * Class ZipkinExporter - implements the export interface for data transfer via Zipkin protocol
+ *
  * @package OpenTelemetry\Exporter
  */
 class ZipkinExporter implements Exporter
 {
 
     /**
-     * @var endpoint to send Spans to
+     * @string $endpoint endpoint to send Spans to
      */
     private $endpoint;
 
     /**
      * Exports the provided Span data via the Zipkin protocol
      *
      * @param iterable<Span> $spans Array of Spans
+     *
      * @return int return code, defined on the Exporter interface
      */
-    public function export(iterable $spans) : int
+    public function export(iterable $spans): int
     {
-        $convertedSpans = [];
+        $converted_spans = [];
```

Comment:
My linter was yelling at me about variables being snake case.  Do we have a strong opinion here?

Reply:
VSCode did this for me.  I agree with you on snake_case rather than camelCase in these particular instances.